static/bidder-params/taboola.json

```diff
@@ -14,6 +14,10 @@
     "tagid": {
       "type": "string"
     },
+    "tagId": {
+      "type": "string",
+      "description": "preferred"
```

Comment:
should also add description for `tagid`. may be we can mark it as deprecated as done in following case

https://github.com/prebid/prebid-server/blob/a1f48b54af8e831927a0f6838f56493fa03efb08/static/bidder-params/appnexus.json#L14

Reply:
change done